railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%%= locale %>">
```

Comment:
That's pretty interesting that it's not documented in the API. I found it in the [internationalization guide](http://guides.rubyonrails.org/i18n.html#the-public-i18n-api) which is not shy about mentioning it.